composer.json

```diff
@@ -5,41 +5,55 @@
     "description": "Blacklist plugin for Sylius.",
     "license": "MIT",
     "require": {
-        "php": "^8.0 || ^8.1 || ^8.2 || ^8.3",
-        "sylius/sylius": "~1.12.0 || ~1.13.0",
-        "doctrine/annotations": "^2.0.0"
+        "php": "^8.2",
+        "dompdf/dompdf": "^2.0",
+        "sylius/calendar": "^0.5.0",
+        "sylius/grid-bundle": "^1.13.0",
+        "sylius/sylius": "~2.0",
```

Comment:
Sylius and PHP versions should be placed on top of "require" section, for better readability.